.github/workflows/actions.yml

```diff
@@ -16,25 +16,83 @@ on:
 jobs:
   build_python:
     name: ${{ matrix.name }}
-    runs-on: ubuntu-18.04
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         include:
-          - name: linting
+          - os: ubuntu-18.04
+            name: linting-ubuntu
+            python-version: 3.6
+
+          - os: ubuntu-18.04
+            name: py36-ubuntu
+            python-version: 3.6
+
+          - os: windows-latest
+            name: py36-windows
             python-version: 3.6
-          - name: py36
+
+          - os: macOS-latest
+            name: py36-mac
             python-version: 3.6
-          - name: py37
+
+          - os: ubuntu-18.04
+            name: py37-ubuntu
             python-version: 3.7
-          - name: py38
+
+          - os: windows-latest
+            name: py37-windows
+            python-version: 3.7
+
+          - os: macOS-latest
+            name: py37-mac
+            python-version: 3.7
+
+          - os: ubuntu-18.04
+            name: py38-ubuntu
             python-version: 3.8
-          - name: py39
-            python-version: 3.9
-          - name: devel
+
+          - os: windows-latest
+            name: py38-windows
+            python-version: 3.8
+
+          - os: macOS-latest
+            name: py38-mac
             python-version: 3.8
-          - name: pypy3
+
+          - os: ubuntu-18.04
+            name: py39-ubuntu
+            python-version: 3.9
+
+          - os: windows-latest
+            name: py39-windows
+            python-version: 3.9
+
+          - os: macOS-latest
+            name: py39-mac
+            python-version: 3.9
+
+          - os: ubuntu-18.04
+            name: pypy3-ubuntu
             python-version: pypy3
+
+          # Disabling due to https://github.com/actions/setup-python/issues/151
```

Comment:
Can you raise an issue to reintroduce this when https://github.com/actions/setup-python/issues/151 is resolved?

Reply:
yep: https://github.com/pytest-dev/pytest-html/issues/358